src/containerapp/azext_containerapp/tests/latest/test_containerappjob_preview_scenario.py

```diff
@@ -0,0 +1,147 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import time
+from time import sleep
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck, live_only)
+from subprocess import run
+
+from .common import (write_test_file, TEST_LOCATION, clean_up_test_file)
+from .utils import create_containerapp_env
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class ContainerappJobPreviewScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(location="eastus", random_name_length=15)
+    def test_containerappjob_preview_environment_type(self, resource_group):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+        aks_name = "my-aks-cluster"
+        connected_cluster_name = "my-connected-cluster"
+        custom_location_id = None
+        try:
+            self.cmd(f'aks create --resource-group {resource_group} --name {aks_name} --enable-aad --generate-ssh-keys --enable-cluster-autoscaler --min-count 4 --max-count 10 --node-count 4')
+            self.cmd(f'aks get-credentials --resource-group {resource_group} --name {aks_name} --overwrite-existing --admin')
```

Comment:
do you need to update kubeconfig?

Reply:
Use AksAndConnectedClusterPreparer to avoid credential leak.